package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@truework/sdk",
-  "version": "0.3.0",
+  "version": "0.5.0",
```

Comment:
You pry know this, but we don't really need to edit this. Semantic release handles versioning with git tags. But also shouldn't hurt anything 🤔 so all good!